packages/spec-test-util/package.json

```diff
@@ -9,7 +9,14 @@
   },
   "homepage": "https://github.com/ChainSafe/lodestar#readme",
   "type": "module",
-  "exports": "./lib/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js"
+    },
+    "./download": {
+      "import": "./lib/downloadTests.js"
+    }
+  },
```

Comment:
Any code which imports `vitest` should be run via `vitest` binary. For download script we need to run it directly with `nodejs` so why have to extract that part as separate export.